doc/sphinx/source/develop/dataset.rst

```diff
@@ -75,14 +76,57 @@ for downloading (e.g. providing contact information, licence agreements)
 and using the observations. The unformatted (raw) observations
 should then be stored then in the appropriate of these three folders.
 
+For each additional dataset, an entry needs to be made to the file 
+`datasets.yml
+<https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/cmorizers/data/datasets.yml>`_.
+The dataset entry should contain:
+
+- the correct ``tier`` information;
+- the ``source`` of the raw data;
+- the ``last_access`` date;
+- the ``info`` that explain how to download the data.
+
+Note that these fields should be identical to the content of the header
```

Comment:
Do we need to keep that info in the header of the cmorizer script?

Reply:
That is a good question. I don't have a strong opinion. Maybe having the info in one place (datasets.yml) is enough. But that needs a bit of extra work to finish the PR.